course/models.py

```diff
@@ -27,9 +27,11 @@ def get_default_academic_year():
 
 class Course(models.Model):
 
+    color = models.CharField(max_length = 7, null = True, blank=True)
     semester_choice = [('autumn', 'Autumn'), ('winter', 'Winter')]
 
     course_code = models.CharField(max_length=20)
+    section = models.CharField(max_length = 15, null = True, blank = True)
```

Comment:
The branch isn't up to date with main on the remote. Rebase or merge latest changes into your branch first.